CHANGELOG-1.0.md

```diff
@@ -0,0 +1,60 @@
+* c01fb64 Call CreateSnapshot to check if snapshot is processed instead of ListSnapshots.
+* e2f1bdc Bumping k8s version to 1.13.0-beta.1.
+* bfba4cb Rename `Ready` to `ReadyToUse` in the `Status` field of `VolumeSnapshot` API object.
+
+## Actions Required
+
+* CSI plugin must support the 1.0 spec. CSI spec versions < 1.0 are no longer supported.
+    * In CSI v1.0, `SnapshotStatus` is removed from `CreateSnapshotResponse`, and is replaced with a Boolean `ReadyToUse`.
+    * The snapshot controller now calls `CreateSnapshot` instead of `ListSnapshots` to check if snapshot is processed after it is cut. Driver maintainer needs to make changes accordingly in the driver.
+    * In CSI v1.0, .google.protobuf.Timestamp is used instead of int64 to represent the creation time of a snapshot. Driver maintainer needs to make changes accordingly in the driver. The creation time is visible to the admin when running `kubectl describe volumesnapshotcontent` to examine the details of a `VolumeSnapshotContent`.
+
+* The renaming from `Ready` to `ReadyToUse` in the `Status` field of `VolumeSnapshot` API object is visible to the user when running `kubectl describe volumesnapshot` to view the details of a snapshot.
+
+* External-provisioner sidecar container which depends on external-snapshotter APIs needs to be updated whenever there is an API change in external-snapshotter. Compatible sidecar container images for external-provisioner and external-snapshotter need to be used together, i.e., both are v1.0.0 or both are v1.0.1.
+
+## Deprecations
+* The following VolumeSnapshotClass parameters are deprecated and will be removed in a future release:
+
+| Deprecated                                          | Replacement                                                                 |
+| ------------------------------------ | --------------------------------------------------- |
+| csiSnapshotterSecretName               | csi.storage.k8s.io/snapshotter-secret-name             |
+| csiSnapshotterSecretNameSpace    | csi.storage.k8s.io/snapshotter-secret-namespace   | 
+
+## Major Changes
+* 7fdc9dd Add classListerSynced for WaitForCacheSync.
+* a32313c Fix initializeCaches bug.
+* d0dde33 Deploy: split out RBAC definitions.
+* 0decedb Update unit tests and disable broken status unit tests.
+* fddc554 Use existing content name if already in snapshot object for static provisioning, otherwise construct new name for content for dynamic provisioning.
+* 96e3bd1 Add `VolumeSnapshotContent` deletion policy.
+* 89a64c9 Fix failed unit tests.
+* ea17039 Add `VolumeSnapshot` and `VolumeSnapshotContent` in Use Protection using Finalizers.
+* 7cd3ef6 Update README to clarify snapshot support status
+* 097a4b8 Use protosanitizer library so secrets will be stripped from the logs.
+* 0cf36ad Adds new reserved prefixed parameter keys which are stripped out of parameter list, and adds deprecation notice for old keys and keep their behavior the same.
+
+# Changelog since v0.4.0
```

Comment:
I think we don't need this since this is the changelog for 1.0.x versions

Reply:
sure.